cluster/service/cluster.go

```diff
@@ -93,10 +155,10 @@ func Start(ctx context.Context, factory service.ClusterCacheFactory, options ...
 			} else {
 				clusterCache = nil
 			}
-			return (clusterCache != nil && started == uint32(1)), err
+			return clusterCache != nil && started == uint32(1), err
```

Comment:
if this code tries to start the cache only once, then maybe https://golang.org/pkg/sync/#Once.Do would be more appropriate ?

Reply:
ok